src/process/status.cr

```diff
@@ -91,6 +91,13 @@ enum Process::ExitReason
   # * On Unix-like systems, this corresponds to `Signal::TERM`.
   # * On Windows, this corresponds to the `CTRL_LOGOFF_EVENT` and `CTRL_SHUTDOWN_EVENT` messages.
   SessionEnded
+
+  # Returns `true` if the process exited abnormally.
+  #
+  # This includes all values except `Normal`.
+  def abnormal?
```

Comment:
Question (non-blocking): is there much value in this method on the Enum? It's not a real value, and it's vague when the Enum is explicit.

I feel like `Process#abnormal_exit?` would be enough.

Reply:
> it's vague when the Enum is explicit.
Could you clarify what you mean by that?

Reply:
Hum, each constant/value is an explicit state, but that new method _looks_ like a constant/value and represents _many_ possible states?